src/python-common/ceph/deployment/service_spec.py

```diff
@@ -605,3 +606,23 @@ def validate_add(self):
         if not self.pool:
             raise ServiceSpecValidationError(
                 'Cannot add ISCSI: No Pool specified')
+
+class HostSpec(ServiceSpec):
```

Comment:
So, is the hostSpec actually a ServiceSpec, or do we need another base class?

I mean, `placement` doesn't make sense for HostSpec.

![image](https://user-images.githubusercontent.com/2574405/81391536-89312580-911d-11ea-8e20-f6a46fb9e41e.png)